projects/ion/src/lib/input/input.component.scss

```diff
@@ -0,0 +1,144 @@
+@import '../../styles/index.scss';
+
+.input {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  background: #fcfcfd;
+  border: 1px solid $neutral-5;
+  border-radius: 8px;
+  padding: 6px 12px;
```

Comment:
You can use the "spacing" function to set the padding:

`padding: spacing(0.75) spacing(1.5);`